tsdoc.json

```diff
@@ -16,10 +16,6 @@
 			"syntaxKind": "inline",
 			"allowMultiple": true
 		},
-
-		// TODO: When following fix is released we can replace below configuration with `extend`
-		// https://github.com/TypeStrong/typedoc/issues/2015
-		// 	"extends": ["typedoc/tsdoc.json"],
```

Comment:
Looks like this [issue](https://github.com/TypeStrong/typedoc/issues/2015) was merged. Do we need to fix our config?

Reply:
Removed `tsdoc` support from the repo (#10) for now as it's not being used